content/en/containers/cluster_agent/admission_controller.md

```diff
@@ -148,6 +182,12 @@ Possible options:
 | `false`          | `admission.datadoghq.com/enabled=true`  | Yes       |
 | `false`          | `admission.datadoghq.com/enabled=false` | No        |
 
+### DogStatsD
+
+To configure DogStatsD clients or other APM libraries that do not support library injection, inject the environment variables `DD_AGENT_HOST` and `DD_ENTITY_ID` by doing one of the following:
```

Comment:
Can we rename the section title to something more generic since we're talking about both dsd clients and apm libraries? Something like `Configuration injection` maybe?